diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x guide.txt origsrc/linuxdoc-tools-0.9.21/Makefile.in src/linuxdoc-tools-0.9.21/Makefile.in
--- origsrc/linuxdoc-tools-0.9.21/Makefile.in	2002-01-28 11:43:59.000000000 -0600
+++ src/linuxdoc-tools-0.9.21/Makefile.in	2006-06-25 16:03:22.158375000 -0500
@@ -87,7 +87,6 @@
 	( cd iso-entities; \
 	  $(MAKE) install )
 endif
-	strip sgmlpre/sgmlpre
 	for ii in sgmlpre/sgmlpre rtf-fix/rtf2rtf; do \
 	  bn=`basename $$ii`; \
 	  $(INSTALL_PROGRAM) $$ii $(bindir)/$$bn; \
@@ -96,8 +95,8 @@
 	@echo "Installing library and support files in $(pkgdatadir)..."
 	if [ ! -d $(pkgdatadir) ]; then mkdir -p $(pkgdatadir); fi
 	chmod 755 $(pkgdatadir)
-	cp -r lib/* $(pkgdatadir)
-	cp VERSION $(pkgdatadir)
+	cp -Lr lib/* $(pkgdatadir)
+	cp -L VERSION $(pkgdatadir)
 	@echo "Installing man pages in $(mandir1) ..." 
 	if [ ! -d $(mandir1) ]; then mkdir -p $(mandir1) ; fi
 	-chmod 755 $(mandir1)
@@ -126,11 +125,11 @@
 	(cd $(pkgdatadir)/dtd;  rm -f linuxdoctr.dtd; ln -s linuxdoctr96.dtd linuxdoctr.dtd)
 	(cd $(bindir); for i in $(progs); do rm -f $$i; ln -s linuxdoc $$i;done)
 	@echo "Installing documentation in $(LINUXDOCDOC) ..."
-	(cd doc; PATH=${PATH}:$(bindir) PREFIX=$(prefix) BINDIR=$(bindir) \
+	(cd doc; PATH="${PATH}:$(bindir)" PREFIX=$(prefix) BINDIR=$(bindir) \
 	 DATADIR=$(datadir) bash Makedoc.sh)
 	if [ ! -d $(LINUXDOCDOC) ]; then mkdir -p $(LINUXDOCDOC); fi
 	chmod 755 $(LINUXDOCDOC)
-	cp -r doc/* $(LINUXDOCDOC)
+	cp -Lr doc/* $(LINUXDOCDOC)
 	rm -f $(LINUXDOCDOC)/Makedoc.sh
 	find $(LINUXDOCDOC) -type d -print | xargs chmod 755
 	find $(LINUXDOCDOC) -type f -print | xargs chmod 644
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make -x guide.txt origsrc/linuxdoc-tools-0.9.21/doc/Makedoc.sh src/linuxdoc-tools-0.9.21/doc/Makedoc.sh
--- origsrc/linuxdoc-tools-0.9.21/doc/Makedoc.sh	2001-12-02 10:49:59.000000000 -0600
+++ src/linuxdoc-tools-0.9.21/doc/Makedoc.sh	2006-06-25 16:06:29.127125000 -0500
@@ -25,7 +25,7 @@
 
 cp -r ${DATADIR}/linuxdoc-tools $TMPDIR
 
-NSGMLS=`which nsgmls`
+NSGMLS=`which onsgmls`
 SGMLSASP=`which sgmlsasp`
 GROFF=`which groff`
 GAWK=`which gawk`
